elements/bulbs-video/elements/summary.test.js

```diff
@@ -71,8 +71,14 @@ describe('<bulbs-video-summary>', () => {
     it('renders <VideoSummaryView>', () => {
       subject = shallow(<BulbsVideoSummary nowPlaying=''/>);
       subject.setState({ video });
+
       expect(subject.equals(
-        <VideoSummaryView video={video} nowPlaying={true}/>
+        <VideoSummaryView
```

Comment:
if you're manually setting these to `null` could you just have `defaultProps` set it to `null`?